Sources/Network/Parse.swift

```diff
@@ -55,4 +56,14 @@ public enum Parse {
 
         return image
     }
+
+    /// Parses empty data
+    ///
+    /// - Parameter data: Raw data
+    /// - Throws: A Parse.Error that can be of type
+    public static func void(data: Data) throws {
```

Comment:
As discussed, it would be nice if we could skip the parsing phase for these types of (Void) Resources, but for now seems to be a good temporary solution 😉

Reply:
Going to ignore this one for now since it seems we are going to break Resources soon and this could probably be discussed then.

Reply:
Will ignore this for now. (see above)